gaphor/SysML/propertypages.py

```diff
@@ -50,7 +50,7 @@ def text_handler(event):
                 buffer.set_text(event.new_value)
                 buffer.handler_unblock(changed_id)
 
-        self.watcher.watch("text", text_handler).subscribe_all()
```

Comment:
I like that we don't have to explicitly subscribe now :+1:

Reply:
In the old setup (Gaphas 2) we needed to. Now we do no longer have to. :smiley: